licenses.d2iq.yaml

```diff
@@ -0,0 +1,436 @@
+    sources:
+      - url: https://github.com/prometheus/prometheus
+        ref: ${image_tag}
+        license_path: LICENSE
+        notice_path: NOTICE
+  - container_image: quay.io/thanos/thanos:v0.17.1
+    sources:
+      - url: https://github.com/thanos-io/thanos
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: docker.io/minio/console:v0.14.8
+    sources:
+      - url: https://github.com/minio/console
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: docker.io/metallb/speaker:v0.9.3
+    sources:
+      - url: https://github.com/metallb/metallb
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: docker.io/mesosphere/kubeaddons-addon-initializer:v0.5.5
+    sources:
+      - url: https://github.com/mesosphere/kubeaddons-extrasteps
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: docker.io/mesosphere/insights:v0.1.6
+    sources:
+      - url: https://github.com/mesosphere/dkp-insights
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.4.1
+    sources:
+      - url: https://github.com/kubernetes/kube-state-metrics
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: docker.io/metallb/controller:v0.9.3
+    sources:
+      - url: https://github.com/metallb/metallb
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: docker.io/nvidia/k8s-device-plugin:v0.9.0
+    sources:
+      - url: https://github.com/NVIDIA/k8s-device-plugin
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: docker.io/minio/minio:RELEASE.2021-02-14T04-01-33Z
+    sources:
+      - url: https://github.com/minio/minio
+        ref: ${image_tag}
+        license_path: LICENSE
+        notice_path: NOTICE
+  - container_image: docker.io/nvidia/dcgm-exporter:2.2.9-2.4.1-ubuntu20.04
+    sources:
+      - url: https://github.com/NVIDIA/dcgm-exporter
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: docker.io/minio/operator:v4.4.10
+    sources:
+      - url: https://github.com/minio/operator
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: k8s.gcr.io/prometheus-adapter/prometheus-adapter:v0.9.1
+    sources:
+      - url: https://github.com/kubernetes-sigs/prometheus-adapter
+        ref: ${image_tag}
+        license_path: LICENSE
+        notice_path: NOTICE
+  - container_image: docker.io/fluent/fluent-bit:1.8.13
+    sources:
+      - url: https://github.com/fluent/fluent-bit
+        ref: ${image_tag}
+        license_path: LICENSE
+  - container_image: docker.io/grafana/grafana:8.3.2
+    sources:
+      - url: https://github.com/grafana/grafana
+        ref: ${image_tag}
+        license_path: LICENSE
+        notice_path: NOTICE.md
+  - container_image: gcr.io/kubecost1/server:prod-1.91.2
```

Comment:
@mhrabovcin 
Im not sure about this. didn't find the repo [but found similar ](https://github.com/mesosphere/kommander-applications/blob/a171ee36ccb6780a61717e80712baf1e5e950eb9/licenses.d2iq.yaml#L249)

Reply:
This looks correct.